samples/cellular/modem_shell/src/ping/icmp_ping.c

```diff
@@ -513,22 +523,43 @@ static void icmp_ping_tasks_execute(struct icmp_ping_shell_cmd_argv *ping_args)
 	enum ping_rai rai;
 	uint32_t ping_t;
 
-	for (int i = 0; i < ping_args->count; i++) {
+	/* All good for ping_args, get the current connection info and start the ping: */
```

Comment:
I would remove ':' from the end of all comments to be consistent

Reply:
Done